src/constants/wallets.ts

```diff
@@ -36,6 +37,7 @@ export enum WalletConnectionType {
   InjectedEip1193 = 'injectedEip1193',
   WalletConnect2 = 'walletConnect2',
   TestWallet = 'TestWallet',
+  Phantom = 'PHANTOM',
```

Comment:
nit: All of the enums from here use a different casing; should this be somewhat consistent?

Reply:
✅